src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -138,3 +138,22 @@ export const getUpdatedUsersStakes = (
 
   return [...usersStakes, newUserStake];
 };
+
+/*
+ * Motion ids are not unique between voting reputation installations. If you had a motion with id of 1
+ * in a previous voting reputation installation, you will get another one after uninstalling and reinstalling
+ * the voting reputation client. So, we only want the one created most recently, since that's the one created
+ * by the currently installed version of the voting reputation extension.
+ */
+
```

Comment:
There is at least one other way this could be handled that comes to mind. 

If we removed all motions from the db that had been initiated by a voting rep extn when it gets uninstalled, we wouldn't have this problem. 

But this means knowing the address of the voting reputation contract that initiated the motion at the time of creating the motion. To my knowledge, this data isn't present on the MotionCreated event, and therefore we'd need to store it somewhere in the block ingestor, either in memory (like in the context object I'm using for listener-removers) or in a file (like the stats file).

I will leave as is unless anyone thinks I should look into the aforementioned alternative, or something else altogether.

Reply:
> If we removed all motions from the db that had been initiated by a voting rep extn when it gets uninstalled, we wouldn't have this problem.

This is no longer the case using the db:
- the actual db id of the Motion is composed of the chain id, extension address and native motion id, so this will be unique across installations
- another thing I wanted to solve with a db, is to not "loose" the "old" motions once a extension instance gets uninstalled, so what you're proposing would prevent that